astropy/coordinates/tests/test_representation.py

```diff
@@ -2099,7 +2099,7 @@ def test_unitphysics(unitphysics):
 
     assph = obj.represent_as(SphericalRepresentation)
     assert assph.lon == obj.phi
-    assert assph.lat == 80*u.deg
+    assert_allclose_quantity(assph.lat, 80*u.deg)
     assert_allclose_quantity(assph.distance, 1*u.dimensionless_unscaled)
```

Comment:
`assert_allclose_quantity` does not exist. How does this not error out? Am I going crazy?

Reply:
Argh nvm... but why?

```python
from astropy.tests.helper import (assert_quantity_allclose as
                                  assert_allclose_quantity)
```